packages/backend-plugin-api/src/wiring/createBackendModule.ts

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import { CONFIG_KEY_PART_PATTERN } from '@backstage/config';
```

Comment:
Let's just duplicate this one instead :p

They just happen to be similar, but I think this deserves it's own pattern. The ID pattern should definitely be a subset of the config key one, but might be even more strict.